pkgs/top-level/beam-packages.nix

```diff
@@ -17,7 +17,7 @@ in
 {
   beamLib = callPackage ../development/beam-modules/lib.nix { };
 
-  latestVersion = "erlang_25";
+  latestVersion = "erlang_27";
```

Comment:
I can't see why we shouldn't do this. Do we need to add a changelog anywhere for the new nixos release stuff?

Reply:
I've decided to postpone this. To limit the scope because I don't want to handle the downstream atm. I want this PR to be a quick one.